src/Raven.Server/Documents/Indexes/IndexStore.cs

```diff
@@ -1167,6 +1167,9 @@ public Index ResetIndex(string name, IndexResetMode indexResetMode = IndexResetM
             if (index == null)
                 IndexDoesNotExistException.ThrowFor(name);
 
+            if (index.Type is IndexType.AutoMap or IndexType.AutoMapReduce)
+                throw new NotSupportedException("Side by side index reset is not supported for auto indexes.");
```

Comment:
Shouldn't it be thrown only for side-by-side reset mode? You could move it to `ResetIndexSideBySideInternal()`

Reply:
also there is an extension method for Type `IsAuto()` IIRC

Reply:
Fixed